src/interfaces/WF/WFIPAddressAddress.ts

```diff
@@ -0,0 +1,9 @@
+/**
```

Comment:
Could this file be renamed to `WFIPAddressSourceOption` please? Just to keep it consistent with the name of the type it defines.